### DISCUSSION TITLE
BMA4XX samples

### DISCUSSION THREAD

**Discussion by jk0010** [zephyrproject-rtos/zephyr#68250] [General]
Are there any plans for BMA4XX samples? Does anyone have any kind of example program?

**Comment by kartben** [comment]
What kind of sample are you looking for?
AFAICT the current BMA4xx driver in Zephyr is pretty generic so you should be able to use e.g. the [sensor_shell](https://docs.zephyrproject.org/latest/samples/sensor/sensor_shell/README.html) sample, or [accel_polling](https://docs.zephyrproject.org/latest/samples/sensor/accel_polling/README.html), [accelerometer_chart](https://docs.zephyrproject.org/latest/samples/modules/lvgl/accelerometer_chart/README.html), ...

**Comment by jk0010** [comment]
I'm looking to read accelerometer data, I'll check those out, thanks for sharing. Currently, I'm getting this error on the serial port. "[00:00:00.500,000] <err> bma4xx: could not read chip_id: -5" trying to figure out why this is happening, this is what I have in my overlay.

```
&i2c1{
	status="okay";
	bma4xx@14 {
		compatible = "bosch,bma4xx";
		reg = <0x14>;
	};
};
```

**Reply by jk0010** [reply]
Back on the error I got earlier. If i have a while loop like this

```
while (1) {
	k_msleep(1000);
}

```
then I will get the error `bma4xx: could not read chip_id: -5`. If I only remove `k_msleep(1000);` then I will have no errors. Is this a possible bug, any ideas here?
